spec/PostgresStorageAdapter.spec.js

```diff
@@ -1,6 +1,17 @@
 const PostgresStorageAdapter = require('../src/Adapters/Storage/Postgres/PostgresStorageAdapter');
 const databaseURI = 'postgres://localhost:5432/parse_server_postgres_adapter_test_database';
 
+const getColumns = (client, className) => {
+  return client.any('SELECT column_name FROM information_schema.columns WHERE table_name = $<className>', { className })
+    .then(columns => {
+      if (!columns.length) {
```

Comment:
What you do in this `.then` block is pointless, i.e. if the condition matches, the you already end up with an empty array.